### PR TITLE
Fix Features page throwing an exception after specific steps are made

### LINKED ISSUE #20777: BO - Features page - an exception is displayed
#### Describe the bug

In the BO > Catalog > Features page, an exception is displayed after some specific steps.

#### Steps to Reproduce

1. Go to BO > Catalog > Features page
2. Create a feature
3. View feature
4. Create values
5. Update feature
6. View updated feature
7. Update 1 value
8. Filter Feature by name => NOK

Other steps to reproduce the issue

1. Create feature
2. View feature
3. Create values
4. Update feature
5. View updated feature
6. Filter value by name
7. Update 1 value => KO

**Screenshots**

image
.

#### Additional information

* PrestaShop version: 1.7.6.7, 1.7.7.x
* PHP version: 7.1

### PR DESCRIPTION
**Pull request by atomiix**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.7.x
| Description?  | Because `AdminFeaturesControllerCore` handles both feature & feature_value forms and a bad `if` statement, when you saved a feature value you were redirected to the list of features. Also, the page could throw an exception if you played with name/value sort buttons in both feature & feature_value pages. This PR fixes it.
| Type?         | bug fix
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | Fixes #20777
| How to test?  | Please check #20777

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/21045)

### CODE REVIEW

**Comment by kpodemski** [comment]
All good, redirection fixed 💪

**Review by kpodemski — DISMISSED** [review]

File: controllers/admin/AdminFeaturesController.php [hunk 1]
```diff
@@ -460,7 +460,7 @@ public function initContent()
     public function initProcess()
     {
         // Are we working on feature values?
-        if (Tools::getValue('id_feature_value')
+        if (Tools::getValue('id_feature_value') && !Tools::getValue('id_feature')
```
Comment (PierreRambaud) [inline controllers/admin/AdminFeaturesController.php]:
```suggestion
        if ((Tools::getValue('id_feature_value') && !Tools::getValue('id_feature'))
```

Comment (atomiix) [reply controllers/admin/AdminFeaturesController.php]: @PierreRambaud Only for better readability, right?

Comment (PierreRambaud) [reply controllers/admin/AdminFeaturesController.php]: Yes, to be sure there is no misunderstanding

**Review by Progi1984 — APPROVED** [review]

**Review by PierreRambaud — APPROVED** [review]

**Comment by Robin-Fischer-PS** [comment]
Hi @atomiix ! Your fix is OK : 
https://drive.google.com/file/d/1HnB3nbvZGW9sg4oxqUBSaVDr_V1zbDz7/view?usp=sharing

Thanks :)

**Comment by Progi1984** [comment]
Thanks @atomiix & @Robin-Fischer-PS

**Pull request merged**